### PR TITLE
54954: document new "Enable SMB1 field" (PR 561 cherry-pick)

### PR DESCRIPTION
**Pull request by Mrt134**
- Replace related images.
- Add note to troubleshooting SMB section.
- HTML build testing: no issues.

### CODE REVIEW

**Comment by dlavigne** [comment]
We already had 490 for master, does this add anything different?

File: userguide/services.rst [hunk 1]
```diff
@@ -1603,6 +1610,13 @@ unless there is a specific need.**
   overhead.
 
 
+.. needs link to smb1 knowledgebase article, when it is published.
+
+Even though the protocol is deprecated and vulnerable, some clients can
```
Comment (wblock) [inline userguide/services.rst]: Suggestion: `` The SMB1 protocol is deprecated and vulnerable. Some older clients still require it.  Before enabling SMB1, see `Why You Should Not Use SMB1 <>`__ for further information.

**Comment by Mrt134** [comment]
Didn't have a choice on the table rework - the automatic merge from the cherry-pick forced me to fix the table.

File: userguide/services.rst [hunk 2]
```diff
@@ -1603,6 +1610,13 @@ unless there is a specific need.**
   overhead.
 
 
+.. needs link to smb1 knowledgebase article, when it is published. Adjust title of link, if necessary.
+
+The SMB1 protocol is deprecated and vulnerable. Some older clients still
+require it. Before enabling SMB1, see `Why You Should Not Use SMB1 <>`__
```
Comment (Mrt134) [inline userguide/services.rst]:
```suggestion
require it. Before enabling SMB1, see `Why You Should Not Use SMB1
<https://www.ixsystems.com/blog/library/do-not-use-smb1/>`__
```

File: userguide/services.rst [hunk 3]
```diff
@@ -1603,6 +1610,14 @@ unless there is a specific need.**
   overhead.
 
 
+.. needs link to smb1 knowledgebase article, when it is published. Adjust title of link, if necessary.
```
Comment (Mrt134) [inline userguide/services.rst]:
```suggestion

```

**Pull request merged**